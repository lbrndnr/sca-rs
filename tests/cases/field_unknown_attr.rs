use scars::Header;

#[derive(Header)]
struct TestHeader {
    #[field(bit_len=4, asdf_qwer=10)]
    pub version: u8
}

fn main() {
    let _hdr = TestHeader {
        version: 1
    };
}