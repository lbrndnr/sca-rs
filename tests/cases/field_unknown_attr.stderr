error: Unknown field attribute
 --> tests/cases/field_unknown_attr.rs:5:25
  |
5 |     #[field(bit_len(4), asdf_qwer(10))]
  |                         ^^^^^^^^^
